lib/git-shell-out-strategy.js

```diff
@@ -61,9 +61,9 @@ const DISABLE_COLOR_FLAGS = [
  * Ex: on Mac ~kuychaco/ is expanded to the specified user’s home directory (/Users/kuychaco)
  * Regex translation:
  * ^~ line starts with tilde
- * ([^/]*)/ captures non-forwardslash characters before first slash
+ * ([^\\\\/]*)[\\\\/] captures non-slash characters before first slash
  */
-const EXPAND_TILDE_REGEX = new RegExp('^~([^/]*)/');
+const EXPAND_TILDE_REGEX = new RegExp('^~([^\\\\/]*)[\\\\/]');
```

Comment:
it's so much more...expandy!  (I really wanted to come up with a better joke about this but couldn't.)